enterprise/backend/src/metabase_enterprise/audit_app/permissions.clj

```diff
@@ -1,28 +1,48 @@
 (ns metabase-enterprise.audit-app.permissions
   (:require
-   [metabase-enterprise.audit-db :refer [default-audit-collection
-                                         default-audit-db-id]]
+   [metabase-enterprise.audit-db
+    :refer [default-audit-collection default-audit-db-id]]
+   [metabase.lib.metadata :as lib.metadata]
    [metabase.models.permissions :as perms]
+   [metabase.models.query.permissions :as query-perms]
    [metabase.public-settings.premium-features :refer [defenterprise]]
+   [metabase.query-processor.store :as qp.store]
    [metabase.shared.util.i18n :refer [tru]]
    [toucan2.core :as t2]))
 
 (def audit-db-view-names
   "Used for giving granular permissions into the audit db. Instead of granting permissions to
    all of the audit db, we query the audit db using the names of each view that starts with v_."
-  ["v_users" "v_content" "v_group_members" "v_alerts_subscriptions" "v_activity"])
+  #{"v_users" "v_content" "v_group_members" "v_alerts_subscriptions" "v_activity"})
```

Comment:
Is it possible to get these names programatically? I think they will have often.

Or a test that verifies that we are covering all v_* views in the database... We now have v_dashboardcard too